### ISSUE TITLE
In scenario 1 and 2, how can one create 2 solo nodes discussing to the same blockchain?

### ISSUE THREAD

**Issue by DavidBruant** [Agoric/cosmic-swingset#77]
While trying to document how people can write contracts on a blockchain with ERTP, i need to do multi-participants contracts. In order to emulate that locally, i would need to:
- setup a blockchain locally (scenarios 1 and 2 already cover this)
- have at least 2 solo nodes discussing with this blockchain
- One method of communicating with each node (maybe different browser tabs connected to localhost servers exposed on different ports or different `.localhost` subdomains)

It's not clear to me how to do that or if that's even currently possible or planned

From what i understand, it's possible to experiment with multi-participant contracts on the Agoric testnet, but i think it'd be useful if people could have a way to test the "multi-participant" aspect of their contract locally before testing on the testnet

**Comment by Chris-Hibbert** [comment]
I talked to @warner, and you're right that this is still unnecessarily hard. I'll [file a bug ](https://github.com/Agoric/cosmic-swingset/issues/78)to make this easier, but the solution won't come very soon. The issue is that the current setup requires each of the clients to setup an initial connection with the chain, and this is currently done in a way that assumes too much common state. There's no obvious work-around that's likely to work now without surfacing new bugs.

When we're doing the same thing here, we set up test cases that use multiple vats interacting with a contract. There are several examples in the ERTP repository under [test/swingsetTests](https://github.com/Agoric/ERTP/tree/master/test/swingsetTests).

In each of these directories, there's a test-foo.js file (which drives the test), bootstrap.js, which has code that runs in the initial vat, and a set of other vat-bar.js files, which represent code running in separate vats. The test code gets references for each of the vats, and can send messages to each, which can start off chains of interaction.
